lib/rules/sort-comp.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
       groups: {
         lifecycle: [
           'displayName',
+          'props',
```

Comment:
If someone is *not* using Flow, and they did in fact have `props = {}` as a class property (setting aside that nobody would ever do that in React), would this sorting category affect it?